spec/std/http/server/server_spec.cr

```diff
@@ -504,6 +509,37 @@ module HTTP
         end
       end
     {% end %}
+
+    it "handles exception during SSL handshake (#6577)" do
+      server = SilentErrorHTTPServer.new do |context|
+        context.response.print "ok"
+        context.response.close
+      end
+
+      server_context, client_context = ssl_context_pair
+      address = server.bind_tls "localhost", server_context
+
+      server_done = false
+      spawn do
+        server.listen
+        server_done = true
+      end
+
+      3.times do
+        3.times do
```

Comment:
Why `3.times`'ing `3.times`, `HTTP::Client.get` can't be included in a `9.times`?

Reply:
Because I wanted multiple wrong calls together.

Reply:
done